kong-0.11.0-0.rockspec

```diff
@@ -27,7 +27,7 @@ dependencies = {
   "luacrypto == 0.3.2",
   "luasyslog == 1.0.0",
   "lua_pack == 1.0.5",
-  "lua-resty-dns-client == 0.6.0",
+  "lua-resty-dns-client == 0.6.1",
```

Comment:
Should we bump this to 0.6.2 now? Considering master is already there.

Reply:
@thibaultcha Done!